src/main/java/com/fdmgroup/forex/repos/OrderRepo.java

```diff
@@ -13,8 +14,12 @@
 @Repository
 public interface OrderRepo extends JpaRepository<Order, UUID> {
 
+	@Query("SELECT o FROM Order o WHERE o.portfolio.user.id = :id")
```

Comment:
Is this annotation a requirement for the operation to work? I was under the impression the JPA repository interface would handle the query in the existing format

Reply:
You're right Elliott
I think perhaps we can just rename the method to something like `findByPortfolio_User_Id` then let JPA handles the actual query.

Reply:
Yes if the objective is to find orders that belong to a given user ID, your line should work:

`findByPortfolio_User_Id`

I have only done a second-order query before (e.g. `findByPortfolio_User`) but in theory it should work this way

Reply:
Resolved by eebb507